flow/Stats.actor.cpp

```diff
@@ -45,21 +45,32 @@ double Counter::getRate() const {
 }
 
 double Counter::getRoughness() const {
-	double elapsed = now() - interval_start;
+	double elapsed = now() - roughness_interval_start;
 	if(elapsed == 0) {
 		return 0;
 	}
 
+	// If we have time samples t in T, and let:
+	// n = size(T) = interval_delta
+	// m = mean(T) = elapsed / interval_delta
```

Comment:
When is the start time of `elapsed`?

Reply:
We're measuring from the last event before our window, see line 48 and 73.